designs/developer/operator-pipelines-47/creation process/index.md

```diff
@@ -0,0 +1,17 @@
+---
+parent: Developer
+version: 4.7
+---
+
+# Improve Pipeline Creation Process
+
+## Description
+When a Pipeline is added during the creation process, there will not be a Build Config associated with the Deployment, Deployment Config, or Knative Service. The Pipeline will be run during creation time unless it cannot be automatically started.
+
+## Not started status
+If a Pipeline has been created but the user has not started it, the Pipeline will have the status of not started. This will be shown on the bottom left decorator in Topology. When the user clicks the not started status decorator, the start modal will open in Topology.
```

Comment:
Out of curiosity, is the lower left decorator always an ellipsis icon?

Reply:
The ellipsis is new and is for pipelines that have not started. The icon is the "build decorator" and displays the status of the build. Other states include new, pending, running, failed and a few more. the icons changes based on the state.